CRITICAL - 2020-06-17 07:24:36 --> Unable to connect to the database.
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(657): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(2455): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Users.php(93): CodeIgniter\Database\BaseBuilder->update()
#3 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Dashboard.php(12): App\Controllers\Users::check()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Dashboard->index()
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-06-17 07:32:34 --> Unable to connect to the database.
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(657): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(2455): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Users.php(93): CodeIgniter\Database\BaseBuilder->update()
#3 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Dashboard.php(12): App\Controllers\Users::check()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Dashboard->index()
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-06-17 07:32:58 --> Unable to connect to the database.
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(657): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(2455): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Users.php(93): CodeIgniter\Database\BaseBuilder->update()
#3 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Dashboard.php(12): App\Controllers\Users::check()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Dashboard->index()
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-06-17 07:36:24 --> Unable to connect to the database.
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(657): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(2455): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Users.php(93): CodeIgniter\Database\BaseBuilder->update()
#3 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Dashboard.php(12): App\Controllers\Users::check()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Dashboard->index()
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-06-17 07:37:06 --> Unable to connect to the database.
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(657): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(2455): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Users.php(93): CodeIgniter\Database\BaseBuilder->update()
#3 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Dashboard.php(12): App\Controllers\Users::check()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Dashboard->index()
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-06-17 07:38:35 --> Unable to connect to the database.
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(657): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(2455): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Users.php(93): CodeIgniter\Database\BaseBuilder->update()
#3 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Dashboard.php(12): App\Controllers\Users::check()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Dashboard->index()
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-06-17 07:39:29 --> Unable to connect to the database.
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(657): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(2455): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Users.php(93): CodeIgniter\Database\BaseBuilder->update()
#3 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Dashboard.php(12): App\Controllers\Users::check()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Dashboard->index()
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-06-17 09:40:42 --> Invalid file: forgot.php
#0 C:\xampp\htdocs\CodeIgniter4\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('forgot.php')
#1 C:\xampp\htdocs\CodeIgniter4\system\Common.php(1087): CodeIgniter\View\View->render('forgot', Array, true)
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Login.php(191): view('forgot', Array)
#3 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Login->forgot()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2020-06-17 09:55:51 --> Unable to connect to the database.
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(657): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\app\Models\UsersModel.php(15): CodeIgniter\Database\BaseConnection->query('select * from u...')
#2 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Login.php(51): App\Models\UsersModel->login(Array)
#3 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(917): App\Controllers\Login->index()
#4 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(403): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(311): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\CodeIgniter4\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
